### ISSUE TITLE
Are these copies or pulled in by the EdX instance?

### ISSUE THREAD

**Issue by effigies** [nipraxis/learn-pages#1]

**Comment by matthew-brett** [comment]
No, I manually convert these to HTML and copy-paste them into the EdX interface.   There's an EdX markdown plugin, but it's not currently installed <https://github.com/open-craft/xblock-html> .  I could install it, it's just I haven't done it yet.

What I'm doing now is writing the files here, then Pandoccing them into HTML, and pasting that into the EdX GUI.

Pandoccing with:

```
pandoc my_markdown_fname.md -t html | pbcopy
```
